### PR TITLE
enh(mc): updated VerneMQ procedure

### PR DESCRIPTION
**Pull request by omercier**
## Description

OT connector update

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 1]
```diff
@@ -5,55 +5,71 @@ title: VerneMQ Restapi
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+VerneMQ est un gestionnaire MQTT évolutif et open source qui connecte l'IdO (IoT), le M2M, le mobile et les applications web.
+Le connecteur de supervision, supervise les Clusters, Listeners, Plugins et sessions en utilisant l'API Rest de VerneMQ.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:9]:
```suggestion
Le connecteur de supervision supervise les Clusters, Listeners, Plugins et sessions en utilisant l'API REST de VerneMQ.
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:9]: plugin ici, sonde après

File: pp/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 2]
```diff
@@ -5,54 +5,70 @@ title: VerneMQ Restapi
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
-## Overview
-
 VerneMQ is a scalable and open source MQTT broker that connects IoT, M2M, Mobile, and web applications
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:8]:
```suggestion
VerneMQ is a scalable and open source MQTT broker that connects IoT, M2M, mobile and web applications.
```

File: pp/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 3]
```diff
@@ -5,54 +5,70 @@ title: VerneMQ Restapi
...
 The VerneMQ Monitoring Connector monitors Clusters, Listeners, Plugins, and sessions using the RestAPI.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:9]:
```suggestion
The VerneMQ Monitoring Connector monitors Clusters, Listeners, Plugins and sessions using the RestAPI.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 4]
```diff
@@ -5,55 +5,71 @@ title: VerneMQ Restapi
...
-| :------------------------------- | :---------------------------------- | :---- |
-| sessions.online.count            | Number of sessions online           | count |
-| sessions.total.count             | Total number of sessions            | count |
+| Métrique              | Unité |
+|:----------------------|:------|
+| sessions.online.count | count |
+| sessions.total.count  | count |
 
 </TabItem>
 </Tabs>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:75]: HTTP or REST API?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 5]
```diff
@@ -5,169 +5,419 @@ title: VerneMQ Restapi
...
+Le support de ces compilations, s'il y en a, est fait par le fournisseur de la distribution associée.
+Leur cycle de publication peut être en retard par rapport aux versions source de VerneMQ.
 
-De plus amples informations sont disponibles sur la documentation officielle du VerneMQ : https://docs.vernemq.com/getting-started
+De plus amples informations sont disponibles sur la documentation officielle de VerneMQ : https://docs.vernemq.com/getting-started
 
 L'API HTTP de VerneMQ est activée par défaut et installe un gestionnaire HTTP sur `http://myvernemq.com:8888/api/v1`.
-L'utilisateur de centreon-engine effectue une connexion à ce système.
-Sur le serveur VerneMQ, vous devez avoir généré un Token avec la commande suivante :
+L'utilisateur **centreon-engine** effectue une connexion à ce système.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:86]: EN and FR are different

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 6]
```diff
@@ -5,169 +5,419 @@ title: VerneMQ Restapi
...
 
-### Métriques collectées
+<Tabs groupId="sync">
+<TabItem value="App-Vernemq-Restapi-custom" label="App-Vernemq-Restapi-custom">
+
+| Alias     | Modèle de service                    | Description            |
+|:----------|:-------------------------------------|:-----------------------|
+| Clusters  | App-Vernemq-Clusters-Restapi-custom  | Contrôle les clusters  |
+| Listeners | App-Vernemq-Listeners-Restapi-custom | Contrôle les listeners |
+| Plugins   | App-Vernemq-Plugins-Restapi-custom   | Contrôle les sondes    |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:29]:
```suggestion
| Plugins   | App-Vernemq-Plugins-Restapi-custom   | Contrôle les plugins    |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md [hunk 7]
```diff
@@ -5,169 +5,419 @@ title: VerneMQ Restapi
...
+Le support de ces compilations, s'il y en a, est fait par le fournisseur de la distribution associée.
+Leur cycle de publication peut être en retard par rapport aux versions source de VerneMQ.
 
-De plus amples informations sont disponibles sur la documentation officielle du VerneMQ : https://docs.vernemq.com/getting-started
+De plus amples informations sont disponibles sur la documentation officielle de VerneMQ : https://docs.vernemq.com/getting-started
 
 L'API HTTP de VerneMQ est activée par défaut et installe un gestionnaire HTTP sur `http://myvernemq.com:8888/api/v1`.
-L'utilisateur de centreon-engine effectue une connexion à ce système.
-Sur le serveur VerneMQ, vous devez avoir généré un Token avec la commande suivante :
+L'utilisateur **centreon-engine** effectue une connexion à ce système.
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-vernemq-restapi.md:86]:
```suggestion
L'utilisateur **centreon-engine** effectue une requête RestAPI à ce système.
```

**Review by cg-tw — APPROVED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Pull request merged**